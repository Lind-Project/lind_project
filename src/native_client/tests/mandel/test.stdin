rpc mandel d(1) d(2) d(3) d(4) * i(0) i(0) i(0)
rpc mandel d(0) d(0) d(0) d(0) * i(0) i(0) i(0)
rpc mandel d(100) d(2) d(3) d(4) * i(0) i(0) i(0)
rpc mandel d(100) d(2) d(3) d(4) * i(0) i(0) i(0)
rpc mandel d(.5) d(.1) d(.5) d(.1) * i(0) i(0) i(0)
