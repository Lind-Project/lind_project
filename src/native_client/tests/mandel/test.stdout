rpc call initiated mandel:dddd:iii
input 0:  d(1)
input 1:  d(2)
input 2:  d(3)
input 3:  d(4)
rpc call complete mandel:dddd:iii
output 0:  i(0)
output 1:  i(0)
output 2:  i(0)

rpc call initiated mandel:dddd:iii
input 0:  d(0)
input 1:  d(0)
input 2:  d(0)
input 3:  d(0)
rpc call complete mandel:dddd:iii
output 0:  i(128)
output 1:  i(0)
output 2:  i(0)

rpc call initiated mandel:dddd:iii
input 0:  d(100)
input 1:  d(2)
input 2:  d(3)
input 3:  d(4)
rpc call complete mandel:dddd:iii
output 0:  i(128)
output 1:  i(0)
output 2:  i(0)

rpc call initiated mandel:dddd:iii
input 0:  d(100)
input 1:  d(2)
input 2:  d(3)
input 3:  d(4)
rpc call complete mandel:dddd:iii
output 0:  i(128)
output 1:  i(0)
output 2:  i(0)

rpc call initiated mandel:dddd:iii
input 0:  d(0.5)
input 1:  d(0.1)
input 2:  d(0.5)
input 3:  d(0.1)
rpc call complete mandel:dddd:iii
output 0:  i(128)
output 1:  i(0)
output 2:  i(0)

